packages/runtime/src/next/api-handler/config/app-router.ts

```diff
@@ -18,19 +19,19 @@ export const argsPattern = [P.instanceOf(Request), P.any] as const
 export async function config({
   req,
   context,
-  apiKey,
+  apiKey: _apiKey,
```

Comment:
I think we can remove the `apiKey` from the internal config params since it's been replaced by the `client`.